apps/mobile/src/screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -55,7 +55,7 @@ function ProfileScreenInner() {
       sharedFollowersFirst: SHARED_FOLLOWERS_PER_PAGE,
       includePosts: isPostEnabled,
     },
-    { fetchPolicy: 'store-or-network', UNSTABLE_renderPolicy: 'partial' }
+    { fetchPolicy: 'store-or-network', UNSTABLE_renderPolicy: 'full' }
```

Comment:
what was the reason for this change?

Reply:
There is an issue that after the user signed up the account with wallet, the own profile screen crash on initial load. 

Its working fine after the screen refreshed. 

My assumption its because of the cache didn't update correctly when new created user logged in.